File: Pkdd.Models/Results/PhysiognomyResult.cs
using Microsoft.EntityFrameworkCore;
using Pkdd.Abstractions.Entity;

namespace Pkdd.Models.Results
{
    [Owned]
    public class PhysiognomyResult : IUpdatable<PhysiognomyResult>
    {
        public double Group1 { get; set; }

        public double Group2 { get; set; }

        public double Group3 { get; set; }

        public double Group4 { get; set; }

        public double Group5 { get; set; }

        public double Group6 { get; set; }

        public double Group7 { get; set; }

        public double Group8 { get; set; }

        public double Group9 { get; set; }

        public double Group10 { get; set; }

        public double Group11 { get; set; }

        public double Group12 { get; set; }

        public PhysiognomyResult Update(PhysiognomyResult source)
        {
            Group1 = source.Group1;
            Group2 = source.Group2;
            Group3 = source.Group3;
            Group4 = source.Group4;
            Group5 = source.Group5;
            Group6 = source.Group6;
            Group7 = source.Group7;
            Group8 = source.Group8;
            Group9 = source.Group9;
            Group10 = source.Group10;
            Group11 = source.Group11;
            Group12 = source.Group12;
            return this;
        }
    }
}

File: Pkdd/pkdd-ng-client/src/app/models/persons/results/physiognomy-result.ts
export interface PhysiognomyResult {
    group1: number;
    group2: number;
    group3: number;
    group4: number;
    group5: number;
    group6: number;
    group7: number;
    group8: number;
    group9: number;
    group10: number;
    group11: number;
    group12: number;
}

export type Group = keyof PhysiognomyResult;

export const emptyPhysiognomyResult = {
    group1: 0,
    group2: 0,
    group3: 0,
    group4: 0,
    group5: 0,
    group6: 0,
    group7: 0,
    group8: 0,
    group9: 0,
    group10: 0,
    group11: 0,
    group12: 0,
}
